lib/modules/dosomething/dosomething_northstar/plugins/openid_connect_client/northstar/OpenIDConnectClientNorthstar.class.php

```diff
@@ -143,10 +143,27 @@ public function retrieveUserInfo($access_token) {
       $base['data']['birthdate'] = strtotime($base['data']['birthdate']);
     }
 
-    if ($base) {
-      return $base['data'];
+    if (! $base) {
+      return null;
```

Comment:
Just switching around the conditionals here so we return `$userinfo` at the bottom & short-circuit otherwise.